### PR TITLE
Update actus to be full manageable

### PR DESCRIPTION
**Pull request by Hacene010**

### CODE REVIEW

File: src/components/HdfTwitter/HdfTwitter.jsx [hunk 1]
```diff
@@ -33,150 +42,64 @@ export default function HdfTwitter() {
       id="Actualités"
       background={colors.hdfTwitter_background_color}
       text={colors.hdfTwitter_text_color}
+      display_actu1={translations.display_actu1}
+      display_actu2={translations.display_actu2}
+      display_actu3={translations.display_actu3}
+      display_actu4={translations.display_actu4}
+      display_actu5={translations.display_actu5}
```
Comment (GeorgesFORTRY) [inline src/components/HdfTwitter/HdfTwitter.jsx]: you need to delete this part

**Review by GeorgesFORTRY — CHANGES_REQUESTED** [review]
a small part to remove

File: src/components/HdfTwitter/HdfTwitter.jsx (hunk 1 above)
Comment (Hacene010) [reply src/components/HdfTwitter/HdfTwitter.jsx]: Yes forgot that into the parent component :/

**Review by GeorgesFORTRY — APPROVED** [review]
ok now

**Review by LoicBrassart — APPROVED** [review]
SnE!

**Pull request merged**